README.md

```diff
@@ -1 +1,144 @@
-# Gamecon
\ No newline at end of file
+<a name="readme-top"></a>
+
+<div align="center">
+  <img src="./images/logo.png" alt="logo" width="140"  height="auto" />
+  <br/>
+
+  <h3><b>Gamecon</b></h3>
+
+</div>
+
+# 📗 Table of Contents
+
+- [📖 About the Project](#about-project)
+  - [🛠 Built With](#built-with)
+    - [Tech Stack](#tech-stack)
+    - [Key Features](#key-features)
+  - [🚀 Live Demo](#live-demo)
+- [💻 Getting Started](#getting-started)
+  - [Prerequisites](#prerequisites)
+  - [Setup](#setup)
+  - [Deployment](#deployment)
+- [👥 Authors](#authors)
+- [🔭 Future Features](#future-features)
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
+- [❓ FAQ (OPTIONAL)](#faq)
+- [📝 License](#license)
+
+# 📖 Gamecon <a name="about-project"></a>
+
+**Gamecon** is a website about the trade fair that is held annually with the same name.
+
+## 🛠 Built With <a name="built-with"></a>
+
+### Tech Stack <a name="tech-stack"></a>
+
+<details>
+  <summary>Client</summary>
+  <ul>
+    <li><a href="https://developer.mozilla.org/en-US/docs/Web/HTML">HTML</a></li>
+    <li><a href="https://developer.mozilla.org/en-US/docs/Web/CSS">CSS</a></li>
+    <li><a href="https://developer.mozilla.org/en-US/docs/Web/javascript">JavaScript</a></li>
+  </ul>
+</details>
+
+### Key Features <a name="key-features"></a>
+
+- **Event Date**
+- **Event Programs**
+- **Event Special Guests**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 🚀 Live Demo <a name="live-demo"></a>
+
+- [Link to Quick Overview of Gamecon](https://www.loom.com/share/872e08af94604713a29378461311bde7?sid=af86d42b-222f-443a-905a-e6f75cd5ec49)
+- [Live link to Gamecon](https://google.com)
+
```

Comment:
- [ ] Kindly make sure to add a live link to your deployed project.

Reply:
Thanks for the review. I have made changes now
![Screenshot 2023-06-30 012756](https://github.com/SnakyMz/Gamecon/assets/115364362/6240d96a-c483-47d1-bd9a-c265acb78383)